runtime/protocol/http/common/src/aws/smithy/kotlin/runtime/http/operation/SdkHttpOperation.kt

```diff
@@ -26,9 +27,12 @@ public class SdkHttpOperation<I, O>(
     internal val serializer: HttpSerialize<I>,
     internal val deserializer: HttpDeserialize<O>,
 ) {
+    /**
+     * The SDK-side request ID for this operation.
+     */
+    public val sdkRequestId: String = Uuid.random().toString()
```

Comment:
I think you can get this off the context already or define an extension to pull it out of the context without needing to also make it a property